src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -438,8 +449,11 @@ export class MeetingManager implements AudioVideoObserver {
     }
   }
 
-  selectAudioInputDevice = async (deviceId: string): Promise<void> => {
-    const receivedDevice = audioInputSelectionToDevice(deviceId);
+  selectAudioInputDevice = async (device: Device | AudioTransformDevice): Promise<void> => {
```

Comment:
Since, we are expanding on the type here and handling the `string` type internally it would not be breaking change right? We also have to update the [`useSelectAudioInputDevice`](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/master/src/hooks/sdk/useSelectAudioInputDevice.tsx) hook as it depends on this method.

Reply:
Updated the `useSelectAudioInputDevice` with the same interface.